src/main/java/dune/task/TaskList.java

```diff
@@ -0,0 +1,190 @@
+package dune.task;
+
+import dune.task.Deadline;
+import dune.task.Event;
+import dune.task.Task;
+import dune.task.ToDo;
+import dune.Storage;
+import dune.DuneException;
+
+import java.time.format.DateTimeParseException;
+import java.util.List;
+import java.util.ArrayList;
+
+/**
+ * Represents a list of tasks. A TaskList has a list of tasks.
+ */
+public class TaskList {
+    private List<Task> tasks;
+
+    private static String before = "Start date must be before end date";
```

Comment:
Perhaps use a name that could explain the usage of the variable (e.g. = dateErrorMessage)?